packages/assert/src/assert.js

```diff
@@ -27,7 +27,7 @@ const globalAssert = globalThis.assert;
 
 if (globalAssert === undefined) {
   throw new Error(
-    `Cannot initialize @agoric/assert, missing globalThis.assert`,
+    `Cannot initialize @agoric/assert, missing globalThis.assert, import 'ses' before '@agoric/assert'`,
```

Comment:
```suggestion
    `Cannot initialize @agoric/assert, missing globalThis.assert. Import 'ses' before '@agoric/assert'`,
```
or
```suggestion
    `Cannot initialize @agoric/assert, missing globalThis.assert; import 'ses' before '@agoric/assert'`,
```

Reply:
yay! one less comma-splice in the world :)